unstructured/partition/pptx.py

```diff
@@ -307,6 +347,14 @@ def _shape_is_off_slide(self, shape: Shape) -> bool:
 class _PptxPartitionerOptions:
     """Encapsulates partitioning option validation, computation, and application of defaults."""
 
+    _PicturePartitionerCls = None
+    """Sub-partitioner used to partition PPTX Picture (Image) shapes.
+
+    This value has module lifetime and is updated by calling the `register_picture_partitioner()`
+    function defined at the module level to replace its value with a pluggable sub-partitioner
+    implementing the `AbstractPicturePartitioner` interface.
```

Comment:
That's a long sentence. Can we break it up into... "This value has module lifetime and is updated by calling the `register_picture_partitioner()` function defined at the module level. `register_picture_partitioner()` replaces `_PicturePartitionerCls` with a pluggable sub-partitioner that implements the `AbstractPicturePartitioner` interface."?

Reply:
Yep, good call. Done.